docs/sharding.md

```diff
@@ -235,6 +263,18 @@ In cross-geo scenario, when a client `/negotiate` with the app server hosted in
 
 ![Normal Negotiate](./images/normal_negotiate.png)
 
+## Dynamic Scale ServiceEndpoints
+<a name="dynamic-scale"></a>
+
+From version 1.5.0, we're enabling dynamic scale ServiceEndpoints for ASP.NET Core version first. So you don't have to restart app server when you need to add/remove a ServiceEndpoint. As ASP.NET Core is supporting default configuration like `appsettings.json` with `reloadOnChange: true`, you don't need to change a code and it's supported by nature. And if you'd like to add some customized configuration and work with hot-reload, please refer to [this](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1).
+
+> Note
+> 
+> Considering the time of connection set-up between server/service and client/service may be a few difference, to ensure no message loss during the scale process, we have a staging period waiting for server connection be ready before open the new ServiceEndpoint to clients. Usually it takes seconds to complete and you'll be able to see log like `Succeed in adding endpoint: '{endpoint}'` which indicates the process completes. But for some unexpected reason like cross-region network issue or configuration inconsistent on different app servers, the staging period will not able to finish correctly. Since fewer things can be done in app server side, we choose to promote the scale as it is. It's suggested to restart App Server when you found the scaling process not working correctly.
+> 
+>  The default value of scale staging period is 5 minutes, and it can be customized by setting the value in [`ServiceOptions.ServiceScaleTimeout`](https://github.com/Azure/azure-signalr/blob/dev/docs/use-signalr-service.md#servicescaletimeout). If you have a lot of app servers, it's good to extend the value a little more.
```

Comment:
`The default value of scale staging period is` => `The default timeout period for the scale is`?

Reply:
`it's good` => `it's suggested`?